docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -135,6 +135,7 @@ default::
 appuio_cloud_memory: true
 appuio_cloud_loadbalancer: true
 appuio_cloud_persistent_storage: true
+appcat_postgresql_vshn_standalone: true
```

Comment:
Let's add this to the known queries on L145/146.

Reply:
Thanks, I totally missed that.